### DISCUSSION TITLE
Adding International Standard Classification of Education (ISCED) property

### DISCUSSION THREAD

**Discussion by KenthJohan** [schemaorg/schemaorg#3043] [Ideas]
Can we have ISCED property?
https://en.wikipedia.org/wiki/International_Standard_Classification_of_Education

Can ISCED be added for https://schema.org/Course?

**Comment by philbarker** [comment]
You can use educationalLevel for ISCED and any other level. Use defined term as the value to specify which classification system you're using. So
```
{
  "@context": "http://schema.org/",
  "@type": "Course",
  "name": "Example",
  "educationalLevel": {
    "@type": "DefinedTerm",
    "name": "241",
    "description": "LOWER SECONDARY EDUCATION: General: Insufficient for level completion or partial level completion, without direct access to upper secondary education",
    "inDefinedTermSet": {
      "name": "ISCED",
      "url": "http://ex.dom.tld/isced" 
    }
  }
}
```

If you think  this doesn't work and ISCED deserves special treatment it could be added.

**Reply by KenthJohan** [reply]
I think ISCED deserves special treatment. It is a world standard. It would make easier for collaboration between different educational providers.

**Reply by philbarker** [reply]
OK, to build a strong case for this it would help to show 
- applications (current or in development) that will make large scale use of the data if it is provided, and 
- sites who are describing courses who would provide that data (e.g. sites that are using schema.org/Course already and have ISCED codes) 

I'm not against proposing properties like ISCED (or EQF level/credit #3044), but very often it is the local codes that are used (NQF level, where N =  national) and there are too many of those to give them a property each, which is why we have the defined term mechanism. Also uptake of the markup that is there has been disappointing so far.

**Comment by stuartasutton** [comment]
@philbarker, while projects may use "local" codes for levels of education, its very supportive of interoperability to crosswalk those those local codes to ISCED (e.g., Credential Engine's CTDL crosswalks its degree levels to ISCED). I think adding it would be useful. 

I am less enamored with including EQF (#3044) . Do we then also include [DQP](https://www.learningoutcomesassessment.org/dqp/). For EQF and DQP (and others) can best be addressed using the DefinedTerm solution you suggest above.

**Reply by KenthJohan** [reply]
https://europa.eu/europass/en/description-eight-eqf-levels
EQF is used in EU. It will be useful for compatibility within EU.
Courses outside EU may want to provide EQF for compatibility with EU.
